crates/holochain_zome_types/src/timestamp.rs

```diff
@@ -371,7 +398,16 @@ impl Sub<Timestamp> for Timestamp {
 #[cfg(feature = "full")]
 impl rusqlite::ToSql for Timestamp {
     fn to_sql(&self) -> rusqlite::Result<rusqlite::types::ToSqlOutput> {
-        Ok(rusqlite::types::ToSqlOutput::Owned(self.0.into()))
+        Ok(rusqlite::types::ToSqlOutput::Owned(
+            self.to_sql_ms_lossy().into(),
```

Comment:
This is weird now because it has changed from secs to ms?

Reply:
I think we decided to use ms as the standard for timestamps in the database, and if so, this makes sense. Do we need anything more than ms precision? And is there anything to be gained from using seconds instead? Opinion @neonphog?

Reply:
Our original discussion had decided that we could fit microseconds safely into the i64 that sqlite uses. If we're happy with that, I'd say we should try to transition everything that way, then we wouldn't need the two separate fields in this Timestamp struct. The only downside would be conversions to javascript... it'd technically work most of the time, but anything > `9007199254740991` can loose precision due to the storage in f64...

Reply:
Luckilly right now we're only in microsecond `1631027863705000`

Reply:
233 years to go until we get to the precision-losing point. And 292445 years until we run out of i64 values.

So shall we standardize on microseconds?

Reply:
That would be my vote... I don't think we ever need nanoseconds if we've got microseconds.